identifiers/country-es/ciudad_autonoma.csv

```diff
@@ -0,0 +1,2 @@
+ocd-division/country:es/autonomous_region:ce,Ceuta
+ocd-division/country:es/autonomous_region:ml,Melilla
```

Comment:
I'd lean towards "region" here instead of "autonomous_region" but I won't block on that.

Reply:
i used "autonomous_region" because of the portugal PR.
Where we have a difference between i.e Lisboa and Acores

ocd-division/country:pt/region:li - Lisboa
ocd-division/country:pt/autonomous_region:ac - Acores
Lisboa is a region in Portugal, but Acores is a autonomous region (no normal region).

So I'd suggest to use "autonomous_region" in this special cases.

Reply:
SGTM.

Reply:
I changes both now to "autonomous_city". 
Sorry for confusion. I used portugal as an example for spain, because of the similar problem.
Ceute and Melilla have in spain an other "political status". So my suggestion would be to have a difference to the other communities, but if you both say "we should use the same ocd type", I can change that again.
@jpmckinney @jdmgoogle

Reply:
I don't know the details for Portugal, so I defer to you, @sguenther85.